api/feat.api/Models/SearchResponse.cs

```diff
@@ -0,0 +1,14 @@
+namespace feat.api.Models;
+
+public class SearchResponse
+{
+    public int Page { get; set; }
+    
+    public int PageSize { get; set; }
+    
+    public long? TotalCount { get; set; }
+
+    public List<Course> Courses { get; set; } = [];
```

Comment:
This will eventually be a list of AI Search results, which will then need running through and populating into SearchResult models via SQL